dev-docs/bidders/relevatehealth.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: bidder
+title: RelevateHealth
+description: Prebid RelevateHealth Bidder Adaptor
+pbjs: true
+biddercode: relevatehealth
+media_types: banner
+usp_supported: true
+coppa_supported: true
+gpp_sids: usstate_all
+deals_supported: true
+floors_supported: true
```

Comment:
Please try to add as much meta data as possible. This helps publishers and ad tech companies vetting your adapter. https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter

- safeframes_ok: true/false
- ortb_blocking_supported: true/partial/false
- dsa_supported: true/false
- tcfeu_supported: true/false

Reply:
@muuki88 Thanks for the suggestions. Made the changes, please review.